### ISSUE TITLE
[Request] IntelliJ IDEA (Community and Ultimate Edition)

### ISSUE THREAD

**Issue by dtrunk90** [terrapkg/packages#595]
**What software are you requesting to be packaged?**
https://www.jetbrains.com/idea/

**Describe the motivation**
It's not packaged for Fedora at all.

**Comment by nothingneko** [comment]
This app is available as a Flatpak but I do think it’s beneficial to have as an RPM

**Comment by dtrunk90** [comment]
Sandboxed apps always cause issues. I personally prefer non-sandboxed apps. IntelliJ has huge problems sandboxed to use the terminal for example.

**Comment by lleyton** [comment]
I agree with @dtrunk90 here. Something like an IDE makes sense outside of a sandbox

**Comment by RokeJulianLockhart** [comment]
Considering that `android-studio` has been packaged, this should be easier.
